docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,93 @@
+Getting around administrator interface
+--------------------------------------
+
+`Edx-enterprise` is integrated with edx-platform Django admin. It can be accessed at
+`$LMS_SERVER_ADDRESS/admin/enterprise` URL.
+
+.. image:: images/enterprise-admin.png
+
+So far, there are two items in the admin interface:
+
+* "Enterprise Customers" - lists and manages Enterprise Customer records.
+* "Enterprise Customer Users" - lists and manages enterprise customer students.
+
+.. image:: images/enterprise-customer.png
+
+Despite providing student management interface in Enterprise Customer Users section, `edx-enterprise` admin provides
+custom (and richer) interface for managing learners associated with particular Enterprise Customer in "Enterprise
+Customer" section, accessible via "Manage Learners" button in the top-right corner of Enterprise Customer edit
+interface.
+
+Manage Learners View
+--------------------
+
+Manage Learners view main goal is to provide means to:
+
+* Associate ("link") and disassociate ("unlink") existing students with Enterprise Customer.
+* Create and remove "future associations" (also known as "Pending Enterprise Customer Users").
+
+In addition, during linking student(s) to Enterprise Customer it is capable of enrolling them into Course or Program.
+
+.. image:: images/manage-learners.png
+
+Integrating with Course Catalog
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order for course enrollment features to work, `edx-enterprise` talks to `Course Catalog IDA`_. Instructions on
+setting up and configuring Course Catalog can be found at corresponding `docs section` of Course Catalog and in
+`edx-enterprise` `Pull Request #7`_ (describes developer setup, but useful for a broader audience).
+
+One particular quirk is that Course Catalog IDA have internal access management, so administrators should access
+`edx-enterprise` admin interface using account registered with Course Catalog IDA. This is covered abovementioned
+docs section and pull request.
+
+.. _Course Catalog IDA: https://open-edx-course-catalog.readthedocs.io/en/latest/getting_started.html
+.. _docs section: https://open-edx-course-catalog.readthedocs.io/en/latest/getting_started.html#lms-integration
+.. _Pull Request #7: https://github.com/edx/edx-enterprise/pull/7
+
+Managing Linked Learners Lists
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Manage Learners view allows for two modes of linking:
+
+* Singular - single user email or username is provided.
+* Bulk - CSV file containing single column of emails is uploaded.
+
+When linking single user by username, `edx-enterprise` tries to find an existing user with that username and fails the
+linking if match was not found. When email is used (both in singular and bulk modes), existing users using that email
+are linked to Enterprise Customer; if email match was not found a "Pending linked learner" record is created for that
+email. When that email is used to register a new user that user is automatically linked with Enterprise customer.
```

Comment:
"~~When~~ If that email address is later used to register an account in the LMS, that user is..."